e2e/src/displayItemDetails/displayItemDetails.e2e-spec.ts

```diff
@@ -0,0 +1,34 @@
+import { browser } from 'protractor';
+import { DisplayItemDetailsPage } from './displayItemDetails.po';
+
+describe('workspace-project App', () => {
+  let page: DisplayItemDetailsPage;
+
+  beforeEach(() => {
+    page = new DisplayItemDetailsPage();
+  });
+
+  it('should display table with 8 action items', () => {
```

Comment:
**SUGGESTION:** That's minor, but the tests should be also documentation and came directly from the requirements, so it's better to name them based on the business scenarios so eg. `should display table with multiple items`. this exact number `8` is not an exact requirement. It's also not the case for the current case (as you're using mocked data) but in the real world you won't know how many items are returned from the api, so it's safer to be greater (or equal) 1.